### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by TweenJRStudio** [MCMrARM/mcpelauncher-linux#404]
I get this Error message when I try to run make -j$(nproc):
/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-pc-linux-gnu/8.1.0/../../../libevdev.so when searching for -levdev
/bin/ld: skipping incompatible /usr/lib/libevdev.so when searching for -levdev
/bin/ld: cannot find -levdev
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/mcpelauncher.dir/build.make:805: mcpelauncher] Error 1
make[1]: *** [CMakeFiles/Makefile2:105: CMakeFiles/mcpelauncher.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

**Comment by jmiahman** [comment]
Assuming this is on Arch and they do not offer a multilib development package for evdev.

**Comment by lukeeey** [comment]
Mrarm uses arch and it runs for him

**Comment by jmiahman** [comment]
Yes if he rolled his own multi-lib package it would :) I would find out, because this isn't the only project I've seen fail with libevdev due to a missing multi-lib package for arch.

**Comment by M3meZ** [comment]
Did you go to the wiki and followed the arch installation instructions?
